src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -27,15 +27,28 @@ void buildSearchConceptsQuery(CloudPlatform platform) {
     assertThat(
         "generated SQL is correct",
         SearchConceptsQueryBuilder.buildSearchConceptsQuery(
-            "condition", "cancer", s -> s, CloudPlatformWrapper.of(platform)),
+            "Condition", "cancer", s -> s, CloudPlatformWrapper.of(platform)),
         equalToCompressingWhiteSpace(
             "SELECT c.concept_name, c.concept_id FROM concept AS c "
-                + "WHERE (c.domain_id = 'condition' "
+                + "WHERE (c.domain_id = 'Condition' "
                 + "AND (CONTAINS_SUBSTR(c.concept_name, 'cancer') "
                 + "OR CONTAINS_SUBSTR(c.concept_code, 'cancer'))) "
                 + "LIMIT 100"));
   }
 
+  @ParameterizedTest
+  @EnumSource(CloudPlatform.class)
+  void buildSearchConceptsQueryEmpty(CloudPlatform platform) {
+    assertThat(
+        "generated SQL for empty search string is correct",
+        SearchConceptsQueryBuilder.buildSearchConceptsQuery(
+            "Condition", "", s -> s, CloudPlatformWrapper.of(platform)),
+        equalToCompressingWhiteSpace(
```

Comment:
I wonder if this should really be `equalToIgnoringWhiteSpace`? It works by compressing the whitespace but that detail isn't important to the caller

Reply:
The IDE tells me equalToIgnoringWhiteSpace is deprecated and to use equalToCompressingWhiteSpace in its place